.travis.yml

```diff
@@ -9,15 +12,20 @@ before_script:
 
 env:
   global:
-    - secure: UDRdz65BGkiFPrTNbMUBaphFD9LJPB2bBOJ5fazgkZqms1VFA6tApPnYHnGrC3YUttW9GJPx2Y937HDZBDzv3CkglqZXo4wx+bBs3O/3LYvGu9jp3hCIpeG5AzqIs1hQvITB8YKtltBpFOptte6UHNkZlNeQl8X1THvT9dxFdCA=
-    - secure: W07zukbZKgNJMNaWUDDtScwAOScqpJfe7tBlh2GFBhVXwmzYKsVB+4qdl9JLFnWvDaHgU8U65M6GueqInjm8dz9rgsn9m5rXDQhx8KfhiRLHZq4tYE9UKBKW+cDK0s4qArUeUwTwi/UtGpfZHKlBrJohueC+dWAjVbNBWmMGQjY=
-    - secure: IT8jMoDJL1QysO4+UT5C1bUPWm6nxjicCCrN21N2/o8OcxZZEGYD/52/y9jWzClAjSmnenVOf+mT8kTvQHOfp95nL2EUPmHVtngzbuqrncRt+zQSIme255XUTksdR1tPI8fZ3nfMW+iZaPA09kLphB8Q7nbarOM6dXZdwaSydPM=
-    - secure: FiticJKWOlcxW4ze9h2sfRjY7RYAgnws87KMzCPEOrduAvpJFqCoGIsnHa7ZAL8LgLSiZOAl68R7lL4R6/lP9BHomqLTygj80ir4ix4HVf1ejIsbOUHxRxu/W+XbiTe5CPphrQhdOydISs2942mey/QtM+2l1w+V7mEGqdVg6s4=
+    - SAUCE_USERNAME=backbone-testing
+    - BROWSER_STACK_USERNAME=ryanroemer2
+    # BROWSER_STACK_ACCESS_KEY
+    - secure: "NFY7urPFNJa2uS25vqSl4YSvp/Ux5vIN6KjJwH3UF6ptwPXBqdYk39aGxoIY2jq5Pm4APzxGOmkSMGWex/+4HrwvKyCW/cFOX6ENxLT2lBb72JEHpkDroZNYPW2zyNsiKDZNqki/XfvuPeNHNOM7Rb3HMPS7xOQJWc8hGO0gJbU="
   matrix:
-    - BB_TESTING_CHECK=1
-    - BB_TESTING_KARMA=ci
-    - BB_TESTING_KARMA=sauce
-    - BB_TESTING_KARMA=bs
```

Comment:
L17-L19 need to be reverted. Only L20 (`BB_TESTING_KARMA=bs`) is replaced with new L20-L22.